draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1845,7 +1849,30 @@ SDP Answer
     </section>
   </section>
 
+  <section anchor="sec-5888" title="Update to RFC 5888" toc="default">
+      <t>
+        This section updates RFC 5888 <xref format="default" pageno="false" target="RFC3261"/>), 
```

Comment:
Should be a reference to RFC 5888, not 3261,

Reply:
Will fix.

Reply:
Looks good